_posts/2021-01-13-voicemeeter-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,385 @@
+---
+layout: post
+title:  "Voicemeeter, an introduction to mixing audio"
+author: flavius
+categories: [ audio ]
+image: ""
+featured: false
+---
+Want to try mixing your audio and eventually have an easier time making adjustments? Don't want to invest in a physical mixer or GoXLR/Elgato Wave with other Digital Mixing software? Come check out this introduction to Voicemeeter, a digital mixing software to help organize your audio.
+
+### - Introduction
+
+Greetings fellow streamers and content creators. Are you interested in taking your audio to the next level but don't have a physical mixer or GoXLR type device such as the GoXLR, GoXLR Mini or Elgato Wave software included with the Elgato Wave microphones? Maybe you are interested in seeing how audio mixing works but don't want to fully commit to a physical device or put tons of money into just to try out? Well, guess what? There are options to try out! While it might not exactly stack up to having physical knobs and sliders to adjust, there is a digital option called Voicemeeter.
+
+You might ask, hey Flav, what is voice meeter? You know what? I can answer that! Voicemeeter is a digital mixing software available as donationware for windows that allows you to pull in different sources of audio and allow for advanced audio routing and configuration. The base version of voicemeeter and banana version are free. The potato version has 30 days and then it periodically asks you for a one time donation to get a key and unlock it fully.
```

Comment:
1. Cut "You know what? I can answer that!" and surround "what is Voicemeeter?" with quotation marks.
2. Capitalize "Windows".
3. Capitalize "Voicemeeter".
4. If you are going to refer to multiple versions of Voicemeeter throughout the article, spell them out explicitly and make sure that if you are going to refer to the shorthand name, tell people that. Do something like, "There are three different versions of Voicemeeter: Voicemeeter, Voicemeeter Banana, and Voicemeeter Potato. Voicemeeter is the most basic and Voicemeeter Potato is the most advanced, with Voicemeeter Banana sitting in a happy middle spot. For brevity, I'll refer to the various versions as Voicemeeter, VMBanana, and VMPotato." Always capitalize the product name when you are referring to it.

Reply:
Complete revamp of the articles since this review.